apis/v1alpha2/local_object_reference_types.go

```diff
@@ -21,7 +21,6 @@ package v1alpha2
 type LocalObjectReference struct {
 	// Group is the group of the referent.
 	//
-	// +kubebuilder:validation:MinLength=1
 	// +kubebuilder:validation:MaxLength=253
 	Group string `json:"group"`
```

Comment:
Would it make sense to backport this change to `v1alpha1` too? Currently, the following is not allowed:

```yaml
apiVersion: networking.x-k8s.io/v1alpha1
kind: Gateway
spec:
  listeners:
  - tls:
      certificateRef:
        name: haproxy-tls
        kind: Secret
        group: "" # 🔥 Validation error, cannot be left empty.
```

which is quite surprising.

Related:
- https://github.com/jcmoraisjr/haproxy-ingress/issues/830

Reply:
Oops, I just noticed the above messages:

> At this point we're only planning to make changes to v1alpha2, v1alpha1 is effectively frozen, so if you revert the v1alpha1 changes the rest LGTM.

> Okay! I reverted the v1alpha1 changes. And so, I reverted the change in examples as well (because they are also v1alpha1).

Never mind my request!